automatic/spotify/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$downloadDir = Get-PackageCacheLocation
+$installer          = 'SpotifyFullSetup.exe'
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'Spotify'
+    file            = Join-Path $downloadDir $installer
+    url             = 'https://download.spotify.com/SpotifyFullSetup.exe'
+    checksum        = '56ACD50DF0D3E4706ABB04F8952536DBB8FAB627A54DF53019E47884AF024D0D'
+    fileType        = 'exe'
+    checksumType    = 'sha256'
+    silentArgs      = '/silent'
+    validExitCodes  = @(0, 1641, 3010)
+}
+
+# Download the installer
+$arguments['file'] = Get-ChocolateyWebFile @arguments
+
+# It doesn't matter what time we choose, we need to start it manually
+schtasks.exe /Create /SC Once /st (Get-Date -Format 'HH:mm') /TN $arguments['packageName'] /TR "'$($arguments['file'])' $($arguments['silentArgs'])" /F 2>$null
```

Comment:
This probably belongs to the helper with something describing the intent. 

Just for reference: https://github.com/chocolatey/chocolatey-coreteampackages/blob/aa05288b177924da94359f27ffc9217541f323cb/automatic/virtualbox/tools/helpers.ps1#L15